static/templates/compose_control_buttons.hbs

```diff
@@ -10,7 +10,4 @@
 <a role="button" class="compose_control_button compose_giphy_link {{#unless giphy_enabled }} hide {{/unless}}" aria-label="{{t 'Add GIF' }}" data-tippy-content="{{t 'Add GIF' }}">
     <i class="compose_gif_icon zulip-icon zulip-icon-gif" tabindex=0></i>
 </a>
-{{#unless hide_drafts_link}}
-<a class="message-control-link drafts-link" href="#drafts" title="{{t 'Drafts' }} (d)">{{t 'Drafts' }}</a>
-{{/unless}}
```

Comment:
@alya just to confirm that you're OK with removing the "Drafts" link in the compose box as part of this change.

Reply:
Yeah, that seems reasonable to me.